webviz_subsurface/plugins/_simulation_time_series/types/derived_ensemble_vectors_accessor.py

```diff
@@ -0,0 +1,47 @@
+import abc
+from typing import Optional, Sequence
+
+import pandas as pd
+
+
+# TODO: Rename?
+# "Possessor"
+# "Accessor"
+# Suggestions:
+# - DerivedEnsembleDataAccessor
+# - DerivedEnsembleData
+# - DerivedEnsembleVectors
+# - DerivedEnsembleVectorsProvider
+# - DerivedVectorsProvider
+# - DerivedVectorsAccessor
```

Comment:
Should we ask the team and decide on name? Or just go for one of them?

Reply:
Would be nice to get some input on the naming, as it feels long. Basically it should describe what it does - accessing vector data, both regular and calculated, for an ensemble (single or delta).

Perhaps the name is good enough as it is?

Reply:
The class name itself is perhaps not being written out in full in so many places (only where initialized), so current (long) name is perhaps good enough 👍

Reply:
Yes, I feel the other possible name would be `DerivedVectorsAccessor`, basically dropping "Ensemble" from the naming